### PR TITLE
Editorial - update links to non-expired versions

### PR DESCRIPTION
**Pull request by m00sey**
***
<a href="https://pr-preview.s3.amazonaws.com/m00sey/vc-data-model/pull/1255.html" title="Last updated on Aug 23, 2023, 3:49 PM UTC (f6557a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1255/9cfb718...m00sey:f6557a3.html" title="Last updated on Aug 23, 2023, 3:49 PM UTC (f6557a3)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3682,7 +3682,7 @@ <h3>Ecosystem Compatibility</h3>
 AnonCreds</a>,
 <a href="https://www.ietf.org/archive/id/draft-mcnally-envelope-02.html">
 Gordian Envelopes</a>, and
-<a href="https://www.ietf.org/archive/id/draft-ssmith-acdc-02.html">
+<a href="https://www.ietf.org/archive/id/draft-ssmith-acdc-03.html">
```
Comment (decentralgabe) [inline index.html:3685]: internet drafts are not stable identifiers - there's no guarantee they'll exist at some point in the future. is there a more stable identifier we could refer to? Perhaps https://datatracker.ietf.org/doc/draft-ssmith-acdc/

File: index.html [hunk 2]
```diff
@@ -3682,7 +3682,7 @@ <h3>Ecosystem Compatibility</h3>
 AnonCreds</a>,
 <a href="https://www.ietf.org/archive/id/draft-mcnally-envelope-02.html">
```
Comment (TallTed) [inline index.html:3683]: As with ACDC --
```suggestion
<a href="https://datatracker.ietf.org/doc/draft-mcnally-envelope/">
```

Comment (m00sey) [reply index.html]: thanks @TallTed

**Review by brentzundel — APPROVED** [review]

**Review by shigeya — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, no changes requested, no objections, merging.

**Pull request merged**